dev/Dockerfile-dev

```diff
@@ -23,7 +23,9 @@ RUN chmod 777 /root
 
 EXPOSE 9002
 EXPOSE 9003
+EXPOSE 5678
 
 ENV OPENSLIDES_DEVELOPMENT 1
 
-CMD [ "python", "-m", "openslides_backend" ]
+# CMD [ "python", "-m", "openslides_backend" ]
```

Comment:
Delete instead of comment. Is the new CMD a full replacement, or can it give issues to other users?

Reply:
With debugpy it starts on Port 5678 the debugserver you can connect with. The rest is a full replacement. Surely, we shouldn't start it on a production environment.